common/src/app/common/types/container.cljc

```diff
@@ -569,23 +570,37 @@
           (gsh/close-attrs? attr val shape-val 1)
           (gsh/close-attrs? attr val shape-val))
 
+        group-not-equal? (and group (not equal?))
+
+
         touched?
-        (and group (not equal?) (not (and ignore-geometry is-geometry?)))]
+        (and group-not-equal? (not (and ignore-geometry is-geometry?)))
```

Comment:
I think we should add `ignored? ` to the and, but I'm not sure

Reply:
I don't understand. What `ignored?`